.cbindgen.toml

```
@@ -0,0 +1,159 @@
+# This is a template cbindgen.toml file with all of the default values.
+# Some values are commented out because their absence is the real default.
+#
+# See https://github.com/mozilla/cbindgen/blob/master/docs.md#cbindgentoml
+# for detailed documentation of every option here.
+
+
+
+language = "C"
+
+
+
+############## Options for Wrapping the Contents of the Header #################
+
+#header = "/* Text to put at the beginning of the generated file. Probably a license. */"
+# trailer = "/* Text to put at the end of the generated file */"
+# include_guard = "my_bindings_h"
+# pragma_once = true
+# autogen_warning = "/* Warning, this file is autogenerated by cbindgen. Don't modify this manually. */"
+include_version = false
```

I would make these changes here

```suggestion
pragma_once = true
autogen_warning = "/* Warning, this file is autogenerated by cbindgen. Don't modify this manually. */"
include_version = true
package_version = true
```

- use `#pragma once` to prevent the header from being loaded multiple times. It is in my opinion supported widely enough to use it rather than define header guards.
- add a warning that this is auto-generated 
- include both the version of cbindgen used to generate the header as well as the version of scryer for which this header is

Note: `package_version` was added by https://github.com/mozilla/cbindgen/pull/939 in cbindgen 0.27.0 but is missing from the template.